### DISCUSSION TITLE
Cannot install to Windows 11 beta

### DISCUSSION THREAD

**Discussion by fabiogras** [nefarius/DsHidMini#124] [Support]
Hi,

Windows 11 beta version 10.0.22000.100 is available for Windows Insiders testing.
When migrating from Windows 10 the driver is failing to be initiated.
Right clicking on the inf file and selecting 'install' does not bring the popup window as described on https://vigem.org/projects/DsHidMini/How-to-Install/

Thanks,

**Comment by nefarius** [comment]
Oh dear, Windows 11 incoming 😅 Please look for `C:\Windows\INF\setupapi.dev.log` and attach it here, thanks!

**Comment by fabiogras** [comment]
[setupapi.dev.log](https://github.com/ViGEm/DsHidMini/files/6911750/setupapi.dev.log)
Attached. There is probably some more stuff going on on the file, I installed SCPToolkit for now.

Thanks!

**Comment by nefarius** [comment]
According to the log the driver installed successfully. Make sure to really wipe every trace of SCP or other conflicting solutions, as per documentation. I can not see anything obvious why it shouldn't work.

**Comment by nefarius** [comment]
Moved away from issues since I got enough positive reports of working on Windows 11 and no bug-indication showed up.
